### ISSUE TITLE
[bug]: `scope` attribute wrongly used on `Getting started - Developers` docs table

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5233]
### 🐞 Describe the bug

On the `Getting Started > Developers` page in the documentation, the `scope="row"` attribute is set on `<td>` elements. The first `<td>` element of each row of this table being a header for the line, it should be a `<th>` instead, then the `scope` attribute would be rightfully used.
https://next.design-system.post.ch/?path=/docs/dfc70c3b-4726-406b-8a41-af5738c89d05--docs
